e2e_tests/integrations/test_huggingface.py

```diff
@@ -161,3 +167,51 @@ def test_run_access_without_callback_configured(self):
 
         with pytest.raises(Exception):
             NeptuneCallback.get_run(trainer)
+
+    def test_log_parameters_indicator(self, environment):
+        with init_run(project=environment.project, api_token=environment.user_token) as run:
+            base_namespace = "custom/base/path"
```

Comment:
What is `base_namespace` for, we have separete test for it in `test_log_with_custom_base_namespace`.

Reply:
In `test_log_with_custom_base_namespace` we're testing that `log(...)` (a user-exposed) method applies the `base_namespace` as a prefix. In this case we're testing the common parameters (an internal only). It's something really different as the correctness of the preceding doesn't impact the following :wink: